Lib/argparse.py

```diff
@@ -2074,6 +2106,10 @@ def consume_optional(start_index):
             # the Optional's string args stopped
             assert action_tuples
             for action, args, option_string in action_tuples:
+                if action.deprecated and option_string not in warned:
+                    self._warning(_("usage of option '%(option)s' is deprecated") %
```

Comment:
I think this would read better (here and throughout):

```suggestion
                    self._warning(_("option '%(option)s' is deprecated") %
```